addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss

```diff
@@ -143,10 +149,15 @@
 .o_AttachmentViewer_headerItemButton {
     cursor: pointer;
 
-    &:hover {
+    &:hover:not(.btn-primary) {
```

Comment:
Still strange rules. Negative should be avoided at all costs in favor of explicit rules.

But this comes from a strange design here, either we should:

- use `button` with `btn btn-...` from bootstrap. Which we try to avoid if possible in mail code in prevision that it will be used in livechat bundle because it must work with the least amount of dependency to other lib to be embed in other sites. But I know that bootstrap button actually has a rather complex style so it's hard to avoid using it.
- or use a custom `div` or `class` such as `o_AttachmentViewer_headerItemButton`, and `o_AttachmentViewer_buttonDownload` like it was done previously here. And then the PR would be to improve on it to add what is missing

but mixing the 2 together will not produce clean CSS.